include/macros.h

```diff
@@ -156,6 +161,48 @@ extern struct GraphicsContext* __gfxCtx;
     }                                                   \
     (void)0
 
+#define GRAPH_ALLOC(gfxCtx, size) Graph_Alloc(gfxCtx, size)
+#define MTXF_TO_MTX(gfxCtx, line, file) Matrix_ToMtx(gfxCtx, line, file)
+#define MATRIX_NEW(gfxCtx, line, file) Matrix_NewMtx(gfxCtx, line, file)
+#define MATRIX_CHECK_FLOATS(mtx, file, line) Matrix_CheckFloats(mtx, file, line)
```

Comment:
`line, file` vs `file, line`

probably something's up

Reply:
Thanks for catching this